packages/meditrak-server/src/routes/importSurveyResponses/assertCanImportSurveyResponses.js

```diff
@@ -57,3 +57,29 @@ export const assertCanImportSurveyResponses = async (
 
   return true;
 };
+
+export const assertSurveyResponseBatchPermissions = async (
+  accessPolicy,
+  models,
+  surveyResponses,
+) => {
+  // Assumes the data has already been validated
+  const entitiesGroupedBySurveyName = {};
```

Comment:
The name you used is still fine! I believe the `grouped` part is implied by the plural in `entities`. Those are implicit conventions used in our codebase:
* `entityBySurveyName` (or more commonly `surveyNameToEntity`) : object of `surveyName` key => `entity` value
* `entitiesBySurveyName`: object of `surveyName` key => `entity[]` value

```suggestion
  const entitiesBySurveyName = {};
```

This is also consistent to the name used in `assertCanImportSurveyResponses()`